dask/utils.py

```diff
@@ -563,3 +563,111 @@ def funcname(func):
         return func.__name__
     except:
         return str(func)
+
+
+def seek_delimiter(file, delimiter, blocksize):
```

Comment:
Is there a reason why this and `read_block` are here rather than in `bytes`? What is the intended scope of that submodule?